src/utils/error.ts

```diff
@@ -1,12 +1,14 @@
 /**
  * This util function detects if the returned object is an error
  * We often need to define those errors within a requested array of multiple instances
- * @param  {any} obj object to evaluate
+ * @param  {unknown} obj object to evaluate
  * @returns {boolean} whether the object is an error
  */
-export const isError = (obj: any) => {
+export const isError = (obj: unknown): boolean => {
   if (!obj || typeof obj !== 'object') {
     return false;
+  } else if (obj) {
+    return 'statusCode' in obj || obj instanceof Error;
   }
-  return 'statusCode' in obj || obj instanceof Error;
+  return false;
```

Comment:
Is there a specific reason why this was changed? I think `else if (obj)` is redundant wrt to `if  (!obj)`? After line 8, we are guaranteed to have an object.

Reply:
Yes you are right, don't know why I changed it like this, maybe I had false-positive errors that are now gone ?